06_structs/code/structs.go

```diff
@@ -4,21 +4,18 @@ import "fmt"
 
 // User is a user type
 type User struct {
-	ID        int
-	Email     string
-	FirstName string
-	LastName  string
+	ID                         int
+	FirstName, LastName, Email string
 }
 
 // // User is a user type
 // type User struct {
-// 	ID                         int
-// 	FirstName, LastName, Email string
+// ID					       int
+// First Name, LastName, Email string
```

Comment:
There should be no space in the `FirstName` property here

Reply:
Fixed!